tests/Unit/NumericalAlgorithms/Spectral/Test_Spectral.cpp

```diff
@@ -157,57 +154,88 @@ namespace {
 
 template <Spectral::Basis BasisType, Spectral::Quadrature QuadratureType,
           typename Function>
-void test_exact_interpolation(const Function& max_poly_deg) {
+void test_interpolation_matrix(const DataVector& target_points,
+                               const Function& max_poly_deg,
+                               const double eps = -1.) noexcept {
```

Comment:
I would rather the default value be 0. than -1. for using the standard `CHECK_ITERABLE_APPROX` below

Reply:
Sure. Another option to make it even more explicit would be to add a bool. Preference?